ctil.cpp

```diff
@@ -4,8 +4,29 @@
 #include "ctil.h"
 namespace cdot
 {
+	bool ctil::ends_with(std::string const & value, std::string const & ending)
+	{
+		if (ending.size() > value.size()) return false;
+		return std::equal(ending.rbegin(), ending.rend(), value.rbegin());
+	}
+
+	void ctil::md_italics_content_update(std::string &input) {
+		size_t found = input.find("*");
```

Comment:
Would it be possible to extend `md_italics_content_update` to also detect for italics underscore syntax?
To allow conversions like `_word_` to `<i>word</i>`?

Reply:
Yes, let me update the `md_italics_content_update` function add underscore syntax check.